custom-recipes/pi-system-transpose/recipe.py

```diff
@@ -36,18 +37,27 @@ def parse_timestamp_and_value(line):
     return date, value
 
 
-def get_datetime_from_string(datetime):
+def get_epoch_from_string(datetime_string):
     try:
-        _ = dateutil.parser.isoparse(datetime)
-        return datetime
+        utc_time = datetime.strptime(datetime_string, "%Y-%m-%dT%H:%M:%SZ")
+        epoch_time = (utc_time - datetime(1970, 1, 1)).total_seconds()
+    except Exception:
+        return None
```

Comment:
We should probably log an error here? I don't think the calling code will be able to handle this?